### DISCUSSION TITLE
Thanks for the Lisp Tutorials.

### DISCUSSION THREAD

**Discussion by aykaramba** [rabbibotton/clog#153] [General]
Dr. Botton, I just wanted to say thank you for the Lisp tutorials.  I just went through all of them and they are the best intro --> training wheels -- > get up and start kind of trying to walk on your own in the shortest amount of time possible.  

All of the books and resources that we are lucky to have for Common Lisp are great and they are of tremendous assistance, however, none of them really get you up and running in a way that is a practical as your tutorials.  I would recommend them to any programming / Lisp beginner.

You work is SERIOUSLY impressive in so many dimensions and greatly appreciated.

Thank you.

**Comment by rabbibotton** [comment]
Thanks, I wish I had the time and language skills to do better, but future reviews will improve them more.
